api/providers/views.py

```diff
@@ -789,3 +793,30 @@ def get_default_queryset(self):
 
     def get_queryset(self):
         return self.get_queryset_from_request()
+
+
+class RegistrationBulkCreate(APIView):
+    parser_classes = [FileUploadParser]
+
+    permission_classes = (
+        drf_permissions.IsAuthenticated,
+        base_permissions.TokenHasScope,
+        CanAddModerator,
+    )
+
+    def put(self, request, *args, **kwargs):
+        provider_id = kwargs['provider_id']
+        file_size_limit = BULK_SETTINGS['DEFAULT_BULK_LIMIT'] * 10000
```

Comment:
I am not familiar with the existing `BULK_SETTINGS`. Should we create a new setting dedicated for bulk upload? Or is this just a quick hack for the draft only?